src/App.vue

```diff
@@ -141,6 +144,27 @@ export default {
       await Promise.all(Object.entries(tokens).map(async ([token]) => this
         .addTokenBalance({ token, balance: await tokenBalance(token, address) })));
     },
+    async getCookiesList(address) {
+      await this.backendAuth('getCookiesList', address)
+        .then((list) => list.forEach((entry) => this.setCookiesListEntry(entry)));
+    },
```

Comment:
move it to store, like
https://github.com/aeternity/superhero-ui/blob/64c318a3843216ea1f0fb9533154fc19985b8def/src/store/index.js#L39-L44